etna/lib/commands.rb

```diff
@@ -2,6 +2,7 @@
 require 'logger'
 require 'rollbar'
 require 'tempfile'
+require 'securerandom'
```

Comment:
Are these commands meant to be in the distributed gem? If so, do you want to bump the version of the gem, so it gets published?

Reply:
and thus I guess you can pin the `etna` gem version down, in whatever backup swarm service is running them?

Reply:
In this case, I'll be using the production image, so it's not imperative to bump the gem.  I think it's ok for them to be available in the gem, they're generally useful as well.  I separate this out from the metis_client however since they have a specific niche use however that I considered outside the norm for metis_client itself.